$OpenBSD: patch-deps_linenoise_linenoise_c,v 1.1 2020/02/11 08:11:13 tb Exp $

Fix an occurrence where char is considered signed on all platforms,
it's not the case on powerpc and arm.

Index: deps/linenoise/linenoise.c
--- deps/linenoise/linenoise.c.orig
+++ deps/linenoise/linenoise.c
@@ -793,7 +793,7 @@ static int linenoiseEdit(int stdin_fd, int stdout_fd, 
 
     if (write(l.ofd,prompt,l.plen) == -1) return -1;
     while(1) {
-        char c;
+        signed char c;
         int nread;
         char seq[3];
 
